### PR TITLE
MVP 1829 project skeleton

### PR DESCRIPTION
**Pull request by andrewmartynov**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+        <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+            <version>1.10</version>
+        </dependency>
+
+        <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
```
Comment (astanovsky) [inline pom.xml]: Do we really need this? Common module has commons-lang3 dependency that being picked up by any dependent project

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+server:
+  port: 8111
+
```
Comment (astanovsky) [inline src/main/resources/application.yml]: Please, add management properties as in new PR for metrics

File: src/main/resources/bootstrap.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    name: pg-source-validation-service
+  cloud:
+    consul:
+      host: consul
+      port: 8500
+      config:
+        format: yaml
+      discovery:
+        instanceId: ${spring.application.name}:${spring.application.instance_id:${random.value}}
+        preferIpAddress: true
```
Comment (astanovsky) [inline src/main/resources/bootstrap.yml]: Please, point health endpoint to /actuator/health

File: src/main/java/com/bigpay/pgsourcevalidation/config/WebSecurityConfig.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package com.bigpay.pgsourcevalidation.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+@EnableWebSecurity
```
Comment (astanovsky) [inline src/main/java/com/bigpay/pgsourcevalidation/config/WebSecurityConfig.java]: Please, move enable annotations to BeansConfig and remove this class

File: src/main/resources/application.yml (hunk 2 above)
Comment (andrewmartynov) [reply src/main/resources/application.yml]: done

File: src/main/resources/bootstrap.yml (hunk 3 above)
Comment (andrewmartynov) [reply src/main/resources/bootstrap.yml]: done

**Pull request merged**